export const changePassword= async(req, res, next)=>
{
    try{

    }
    catch(err)
    {
        console.log(err);
        
    }
}